packages/scxa-marker-gene-heatmap/__test__/CellTypeMarkerGeneHeatmap.test.js

```diff
@@ -1,92 +1,75 @@
 import React from 'react'
-import Enzyme from 'enzyme'
-import {shallow} from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
-
-import '@babel/polyfill'
+import { render, screen } from '@testing-library/react'
 import MarkerGeneHeatmap from '../src/MarkerGeneHeatmap'
 
-Enzyme.configure({ adapter: new Adapter() })
-
 describe(`MarkerGeneHeatmap`, () => {
   test(`creates plot lines for every cell type`, () => {
```

Comment:
Shall we change `test` to `it` to unify the usage of a test definition at least inside a component?
Thanks

Reply:
I modified all test to `test` insteadly. The test and it functions in Jest serve the same purpose but differ in their naming conventions and how they impact the readability of test descriptions. The choice between test and it is about readability rather than functionality. `test` is for more general tests, like UI or snapshots, `it` is more for unit functional tests.